client/src/pages/ManageBookingPage/Calendar.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import './Calendar.scss';
+// import { MdChevronLeft, MdChevronRight } from 'react-icons/md';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import moment, { Moment as MomentTypes } from 'moment';
+
+function Calendar() {
```

Comment:
cool! This is an interesting solution!

Reply:
Thank you! but it's hard one to control though